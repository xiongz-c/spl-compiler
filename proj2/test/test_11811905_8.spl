int g1;

int func(int p1, float p2) {
    g1 = 2.0;
    return 0; 
}

int test_11811806_8() {
    float v1 = 1.0;
    return func(g1, v1) + 2.0;
} 