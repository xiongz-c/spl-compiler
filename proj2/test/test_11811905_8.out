Error type 5 at Line 4: unmatching type on both sides of assignment
Error type 8 at Line 10: incompatiable return type
